### DISCUSSION TITLE
Tabs larger than page width

### DISCUSSION THREAD

**Discussion by Tomperez98** [saadeghi/daisyui#2750] [Q&A] [answered]
Laptop
<img width="1467" alt="image" src="https://github.com/saadeghi/daisyui/assets/72174660/4322667d-2617-4060-b186-2c6155ebcd15">

Iphone SE
<img width="510" alt="image" src="https://github.com/saadeghi/daisyui/assets/72174660/ab93b79b-307c-4647-8d3c-125f87b7fb39">

I'm really new in front end programing. Any idea how to fix this?

**Comment by saadeghi** [comment, marked as answer]
It's a horizontal tab layout so if there are many tabs, it won't fit the space. If you have to use tabs for here, you can do something like this:  
https://play.tailwindcss.com/aVWcHBDCWr?size=370x286
```html
<div class="overflow-x-auto">
<div role="tablist" class="tabs tabs-bordered">
  <input type="radio" name="my_tabs_1" role="tab" class="tab whitespace-nowrap" aria-label="Tab 1" />
  <div role="tabpanel" class="tab-content p-10">Tab content 1</div>

  <input type="radio" name="my_tabs_1" role="tab" class="tab whitespace-nowrap" aria-label="Tab 2" checked />
  <div role="tabpanel" class="tab-content p-10">Tab content 2</div>

  <input type="radio" name="my_tabs_1" role="tab" class="tab whitespace-nowrap" aria-label="Tab 3 (I'm writing a really really long text for this tab so we can see the overflow )" />
  <div role="tabpanel" class="tab-content p-10">Tab content 3</div>
</div>
</div>
```
`overflow-x-auto` creates a horizontal scroll, `whitespace-nowrap` forces the tab names to stay in one line even if there's no space.

Let me know if you have a question.
